examples/catalog/config.json

```
@@ -13,5 +13,7 @@
       "url": "<LINK TO EXPLORERS HUB TOPIC>"
     }
   },
-  "whatsNew": "You can now sort your cloud dependencies with a great new look using bullets and /n"
+  "whatsNew": "You can now sort your cloud dependencies with a great new look using bullets and /n",
+  "categoryTerms": ["categoryTerm1", "categoryTerm2"],
```

What do you think of making one of these an actual categoryTerm
```suggestion
  "categoryTerms": ["browser agent", "categoryTerm2"],
```

---

For sure! I wasn't quite sure what to put so thank you :)